.dockerignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
Is this still needed? What effect does it have?

Reply:
No need, it is removed. 71172291bf8b83e22a640dc120baa19947de6cee